cdc/entry/mounter.go

```diff
@@ -480,13 +481,23 @@ func formatColVal(datum types.Datum, tp byte) (
 		v, err := datum.GetBinaryLiteral().ToInt(nil)
 		const sizeOfV = unsafe.Sizeof(v)
 		return v, int(sizeOfV), "", err
-	case mysql.TypeString, mysql.TypeVarString, mysql.TypeVarchar,
-		mysql.TypeTinyBlob, mysql.TypeMediumBlob, mysql.TypeLongBlob, mysql.TypeBlob:
-		b := datum.GetBytes()
-		if b == nil {
-			b = emptyBytes
+	case mysql.TypeString, mysql.TypeVarString, mysql.TypeVarchar:
+		v := datum.GetString()
+		const sizeOfV = unsafe.Sizeof(v)
```

Comment:
The actual data of string is in heap, so I think size should be `sizeOfString(v)`